command.go

```diff
@@ -903,8 +903,8 @@ func (c *Command) ExecuteC() (cmd *Command, err error) {
 			c = cmd
 		}
 		if !c.SilenceErrors {
-			c.Println("Error:", err.Error())
-			c.Printf("Run '%v --help' for usage.\n", c.CommandPath())
+			c.PrintErrln("Error:", err.Error())
+			c.PrintErr(fmt.Sprintf("Run '%v --help' for usage.\n", c.CommandPath()))
```

Comment:
Why not use `PrintErrf()`, which you've fixed further below?

Reply:
Seems legit. Will amend accordingly.